typehints/micropython/machine.pyi

```diff
@@ -265,6 +273,12 @@ Usage Model::
         """
         ...
 
+    class board(Enum):
```

Comment:
I cannot find this `board` attribute of `machine.Pin` in the MicroPython documentation. Could you please share a reference to the page that describes this API?

Reply:
Here is the documentation for the board. I am not sure why it is listed under the pyb.Pin class because my program runs in from machine.Pin so it should be available on all boards but with different available values.
https://docs.micropython.org/en/latest/pyboard/library/pyb.Pin.html

I added a second commit to my pull request. for this to work you need to import Enum. Enum itself is not available in micopython but I shows up properly in pycharm.

Reply:
Here is the dir output for machine.Pin:

>>> dir(machine.Pin)
['__class__', '__name__', 'AF1_TIM1', 'AF1_TIM2', 'AF2_TIM3', 'AF2_TIM4', 'AF2_TIM5', 'AF3_TIM10', 'AF3_TIM11', 'AF3_TIM8', 'AF3_TIM9', 'AF4_I2C1', 'AF4_I2C2',
 'AF5_SPI1', 'AF5_SPI2', 'AF7_USART1', 'AF7_USART2', 'AF7_USART3', 'AF8_UART4', 'AF8_USART6', 'AF9_CAN1', 'AF9_CAN2', 'AF9_TIM12', 'AF9_TIM13', 'AF9_TIM14', 'A
F_OD', 'AF_PP', 'ALT', 'ALT_OPEN_DRAIN', 'ANALOG', 'IN', 'IRQ_FALLING', 'IRQ_RISING', 'OPEN_DRAIN', 'OUT', 'OUT_OD', 'OUT_PP', 'PULL_DOWN', 'PULL_NONE', 'PULL_
UP', 'af', 'af_list', 'board', 'cpu', 'debug', 'dict', 'gpio', 'high', 'init', 'irq', 'low', 'mapper', 'mode', 'name', 'names', 'off', 'on', 'pin', 'port', 'pu
ll', 'value']